dipy/reconst/tests/test_dti.py

```diff
@@ -772,13 +772,13 @@ def test_adc():
     pdd0 = dtifit.evecs[0, 0, 0, 0]
     sphere_pdd0 = dps.Sphere(x=pdd0[0], y=pdd0[1], z=pdd0[2])
     npt.assert_array_almost_equal(
-        dtifit.adc(sphere_pdd0)[0, 0, 0], dtifit.ad[0, 0, 0], decimal=5
+        dtifit.adc(sphere_pdd0)[0, 0, 0], dtifit.ad[0, 0, 0], decimal=4
```

Comment:
I had to reduce precision, otherwise test_adc was failing, it was not able to pass with `5` decimal precision.

Reply:
ok